<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatDialog</class>
 <widget class="QDialog" name="chatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chatDialog</string>
  </property>
  <widget class="QPushButton" name="pbLogin">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登 录</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbUserName">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>137</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbUserPass">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>177</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtUserName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtUserPass">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbTitle">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>聊天</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbWarning">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>用户名或密码不正确！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRegister">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注 册</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
